readme.md

```diff
@@ -23,8 +23,8 @@ This is possible because [we use the Parcel library](https://en.parceljs.org/scs
 
 ❗️ Replace `<your_account>` with your GitHub username and copy the links to the `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_product-cards/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_product-cards/report/html_report/)
+- [DEMO LINK](https://<captain-cactus-ua>.github.io/layout_product-cards/)
+- [TEST REPORT LINK](https://<captain-cactus-ua>.github.io/layout_product-cards/report/html_report/)
```

Comment:
Fix these links and update it in the PR description, now it's wrong 
<img width="216" alt="image" src="https://github.com/user-attachments/assets/1290be46-f712-42c3-9aa3-9ea8a72f21a6">
```suggestion
- [DEMO LINK](https://captain-cactus-ua.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://captain-cactus-ua.github.io/layout_product-cards/report/html_report/)
```